lib/compile-exports.js

```diff
@@ -11,19 +11,26 @@ module.exports = function compileExports(result, importItemMatcher, camelCaseKey
     return "";
   }
 
+  var symbols = [];
   var exportJs = Object.keys(result.exports).reduce(function(res, key) {
     var valueAsString = JSON.stringify(result.exports[key]);
     valueAsString = valueAsString.replace(result.importItemRegExpG, importItemMatcher);
-    res.push("\t" + JSON.stringify(key) + ": " + valueAsString);
+
+    camelCaseKey = camelCase(key);
 
     if (camelCaseKeys === true) {
-      res.push("\t" + JSON.stringify(camelCase(key)) + ": " + valueAsString);
+      symbols.push(camelCaseKey + ": " + camelCaseKey);
     } else if (camelCaseKeys === 'dashes') {
-      res.push("\t" + JSON.stringify(dashesCamelCase(key)) + ": " + valueAsString);
-    }
+      camelCaseKey = dashesCamelCase(key);
+      symbols.push(camelCaseKey + ": " + camelCaseKey);
+    }     
+
+    res.push("export const " + camelCaseKey + " = " + valueAsString);
+    symbols.push("'" + key + "': " + camelCaseKey);
 
     return res;
-  }, []).join(",\n");
+  }, []);
+  exportJs.push("export default { " + symbols.join(", ") + " };");
```

Comment:
It should not default export the key, but export each symbol separate. Best emit an

``` js
export var className = "class-name";
```

per class name.

The CSS itself should be exported via default export.

``` js
export default css;
```